### PR TITLE
Script functionality and changed name of NewLevel

### PR DESCRIPTION
**Pull request by farskw0w**
Created new game controller and plugged in scripts from template. Music plays and tokens/enemies are able to be interacted with.

Changed the name of NewLevel to Seamount. ---Unsure if changing the name of the scene will cause issues once merged into main.

Seamounts — undersea mountains formed by volcanic activity — were once thought to be little more than hazards to submarine navigation. Today, scientists recognize these structures as biological hotspots that support a dazzling array of marine life.

### CODE REVIEW

**Comment by CharmedSatyr** [comment]
@farskw0w Thanks for coming up with such a thoughtful name and background for the environment! Love it!
This pretty much works as expected. We can tie in the new score/title/victory/build settings in another PR. I can try to do it tonight, just a few parameters to update, and then put up a new build on itch.io.

A couple issues:
1. I get a null reference exception when starting the scene. That is because your Meta Game Controller doesn't have a Main Menu supplied. You can probably just go to the panel for Game Controller and uncheck the script Meta Game Controller. We're not really using it right now and can add a value for Main Menu later if we need to.
2. I don't get a victory interaction when I get to the little house at the end. (Look for the VictoryZone object and associated scripts.) Please do port that over so we can hook into it for the victory scene.

**Comment by farskw0w** [comment]
image
resolving this now, I changed a setting in the build settings to try and make Seamount the default level, will revert this change

**Comment by farskw0w** [comment]
Seems to have no conflicts now, should be good?

File: ProjectSettings/TimelineSettings.asset
Comment (CharmedSatyr) [inline ProjectSettings/TimelineSettings.asset:1]: This was auto-generated, but do you know what caused it?

**Review by CharmedSatyr — APPROVED** [review]

**Pull request merged**